### ISSUE TITLE
Error handling rooms with non-Latin names (actually `fname`s)

### ISSUE THREAD

**Issue by drygdryg** [verdigado/rocketchat2matrix#28]
The commit https://github.com/verdigado/rocketchat2matrix/commit/56da775d0179391ac3dfd380b2944c7e3fdf7a3c introduces a problem with rooms with non-Latin names: `fname` can contain non-Latin characters, but `name` cannot. Matrix-synapse prohibits the use of non-Latin characters in the room alias.
```
info: Parsing room with ID: Th3H3SwayaAWxjbqweEspqD7FSz2ZhJZSJ
error: Error during request: Request failed with status code 400
error: Request: POST /_matrix/client/v3/createRoom
error: Response: 400 {"errcode":"M_UNKNOWN","error":"Invalid characters in room alias"}
```
I think `room.room_alias_name` should always be `name`, and `room.name` should be `fname` (falling back to `name`):
https://github.com/verdigado/rocketchat2matrix/blob/698062c7f15e3786320e20066bbf0aee05f8945b/src/handlers/rooms.ts#L91-L93
```typescript
  if (rcRoom.fname || rcRoom.name) {
    room.name = rcRoom.fname || rcRoom.name
    if (rcRoom.name) {
      room.room_alias_name = rcRoom.name
    }
  }
```
Alternatively, we can slugify `fname` before using it as a room alias using https://github.com/sindresorhus/slugify.

**Comment by HerHde** [comment]
> `fname` can contain non-Latin characters, but `name` cannot
In Rocket.Chat. But in Matrix `name` can contain non-Latin characters, but `room_alias_name` cannot. Alright.

**Comment by HerHde** [comment]
Implemented like you proposed in 9358ba200ed7d0a67a0ae7b5f6613b2ceb145c66.
Thank you, again!
